docs/pyos-package-dashboard-workflow.md

```diff
@@ -0,0 +1,318 @@
+# pyOpenSci Package Dashboard Development Workflow
+
+## Overview
+This guide covers the development workflow for the package-activity.qmd dashboard located in metrics/pyos-packages/. This dashboard specifically focuses on **identifying packages that have been inactive for 6+ months** to help the pyOpenSci team prioritize maintenance efforts.
+
+## File Purpose
+The `package-activity.qmd` creates a focused dashboard that shows:
+- **Inactive Package Identification** - Primary focus on packages needing attention (6+ months without commits)
+- **Package Activity Timeline** - All packages sorted by last commit date for context
+- **Maintenance Prioritization** - Clear separation between active and inactive packages
+- **Summary Statistics** - Quick overview of package health across the ecosystem
+
+## Prerequisites
+
+### System Requirements
+- **Python**: 3.9+ (3.10+ recommended for full functionality)
+- **Quarto**: Installed and accessible via command line
+- **Git**: For version control
+
+### Required Python Packages
+```bash
+pandas>=1.5.0
+altair>=4.2.0
+plotly>=5.0.0
+itables>=1.0.0
+jupyter>=1.0.0
+pyarrow>=10.0.0
+```
+
+### Data Dependencies
```

Comment:
There is some good information in this file but we should move it to DEVELOPMENT.MD file rather than this file which seems to be specific to a single page in the guide. the information about where the data live and how they are used is super important. but needs to be organized in a normal open source documentation strcuture (given we don't use sphinx here). 

This page below overviews CONTRIBUTING and DEVELOPMENT guides
https://www.pyopensci.org/python-package-guide/documentation/repository-files/intro.html